test/verify/check-kdump

```diff
@@ -17,23 +17,43 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+import packagelib
 import testlib
 from lib.constants import TEST_OS_DEFAULT
 
 
+def run_ansible_playbook(machine, browser, allow_failure=False):
+    browser.click("#kdump-automation-script")
+    ansible_script_sel = ".automation-script-modal .pf-v5-c-modal-box__body section:nth-child(2) textarea"
+    machine.execute("mkdir -p roles/kdump/tasks")
+    machine.write("roles/kdump/tasks/main.yml", browser.text(ansible_script_sel))
+    with browser.wait_timeout(120):
+        # Redirect to stderr as this can fail, if so we want logs
+        machine.execute(f"ansible -m include_role -a name=kdump localhost {'>&2' if allow_failure else ''}", check=not allow_failure)
```

Comment:
You can also say `, stdout=None)`. I think we always want the output -- it's nice for running locally and debugging, and run-tests hides the output of successful tests anyway.